angular-frontend/src/app/app-routing.module.ts

```diff
@@ -1,52 +1,115 @@
 import { Component, NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import { PrivateAuthGuard} from './auth/auth-guards/private-auth.guard';
 import { CompanyInvoiceListComponent } from './components/company-invoice-list/company-invoice-list.component';
 import {CompanyListComponent} from "./components/company-list/company-list.component"
 import { CompanyComponent } from './components/company/company.component';
 import { CreateCompanyComponent } from './components/create-company/create-company.component';
 import { CreateInvoiceComponent } from './components/create-invoice/create-invoice.component';
+import { HomeComponent } from './components/home/home.component';
 import { InvoiceItemComponent } from './components/invoice-item/invoice-item.component';
 import { InvoiceListComponent } from './components/invoice-list/invoice-list.component';
 import { InvoiceUpdateComponent } from './components/invoice-update/invoice-update.component';
+import { LoginComponent } from './components/login/login.component';
+import { RegisterComponent } from './components/register/register.component';
 import { TaxReportComponent } from './components/tax-report/tax-report.component';
+import { UserListComponent } from './components/user-list/user-list.component';
 
 const routes: Routes = [
+{
+  path: "",
+  component: HomeComponent
+
+},
 {
   path: "companies",
-  component: CompanyListComponent
```

Comment:
There is a possibility to create "dummy" path with canActivate and put every other path as its child - try to find a way for that (kind of challange for you :) ) If you will have some problem, I can help